index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css" rel="stylesheet" />
+  <title>Consciousness Explained</title>
+</head>
+
+<body>
+  <header>
+    <nav class="second-nav">
+      <a href="#" aria-label="bars"><i class="fa fa-bars"></i></a>
+    </nav>
+  </header>
+  <section class="first-page d-flex">
+    <div>
+    <h2 class="red welcome">"Hello! Welcome!"</h2>
+    <h1 class="red comm">CONSCIOUSNESS EXPLAINED SUMMIT 2025</h1>
+  </div>
+    <div class="bg-grey">
+      <p>A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries is taking place in October, in Korea.</p>
+    </div>
+    <h2 class="black date">2025.10.15(THU)~16(FRI)</h2>
+    <p class="at">@National Museum of America,Center Nabi</p>
+  </section>
+  <section class="main-prog d-flex">
+    <h5 class="program-head">Main Program</h5>
+    <div class="program-line red"></div>
+    <div class="list d-flex">
+
+      <ul class="program">
+        <li>
+          <i class="fas fa-clipboard"></i>
+          <h2 class="red">Lecture</h2>
+          <p>Listen to the speakers from various countries about the messeges of sharing and opening</p>
+        </li>
+        <li>
+          <i class="fas fa-hammer"></i>
+          <h2 class="red">Workshop</h2>
+          <p>Try creating your own work using open source license rather than just watching art</p>
+        </li>
+        <li>
+          <i class="fas fa-eye"></i>
+          <h2 class="red">CC Exhibition</h2>
+          <p>Appreciate varioous creations applying CE license of artists,organized from Art Center Nabi.</p>
+        </li>
+        <li>
+          <i class="fas fa-comments"></i>
+          <h2 class="red">Forum</h2>
+          <p>Have the time to share your thoughts and opinions with experts for each topic</p>
+        </li>
+        <li>
+          <i class="fas fa-anchor"></i>
+          <h2 class="red">Ignite</h2>
+          <p>Get opportunities to network with CE affiliates around the world, also after the summit</p>
+        </li>
+      </ul>
+      <button type="button" class="red join">Join CE Gloabal Summit 2025</button>
+    </div>
+  </section>
+  <section class="speakers d-flex">
+    <h2 class="program-head fs">Featured Speakers</h2>
+    <div class="program-line red fs-li"></div>
+    <ul class="li-of-speakers">
+      <li>
+        <img src="images/download.jpeg" alt="">
+        <div class="line-t"></div>
+        <div>
+          <h2>Daniel Dennett</h2>
+          <span>American philosopher</span>
+          <div class="break"></div>
```

Comment:
- [ ] The speaker section is expected to resemble the given design kindly ensure you include the background image seen in the original.

| Original | Yours |
| --------- | ---------- |
| <img src="https://user-images.githubusercontent.com/33323077/132184630-85a73a54-b42e-4a7a-9783-fcb722bbc5d7.png" height="250px"/> | <img src="https://user-images.githubusercontent.com/33323077/132184860-75c97e32-0a6b-4674-8abe-0790c9aa755d.png" height="250px"/> |

Reply:
where can I get that image from?

Reply:
You can create it with CSS. Here's an example: 

<img src="https://user-images.githubusercontent.com/33323077/132193699-e43d6b99-032f-4f94-a51a-dd90802b62dd.jpg" height="300px"/>

Reply:
fixed